src/logger.ts

```diff
@@ -1,5 +1,7 @@
 import pino from 'pino';
 
-const logger = pino();
+const logger = pino({
+  level: process.env.LOG_LEVEL ?? 'silent',
```

Comment:
I changed the default log level to silent in an epic battle against noisy tests, but forgot to change it back to info.

Also we might consider detecting if Jest `--silent` is enabled and having that override log level when set.  Here is an example of how to do that: https://github.com/chriszs/tv-kitchen/blob/ff61ea8611d092740e38a020b06d7aff4cd979f1/lib/utils/jest.setup.js#L9

Reply:
What was the noise during tests? From the pino-http request logger? Or did pino somehow get auto-changed to `debug` level during the tests? I am unfortunately a fan of verbosity. Perhaps it is the Java that is in me. I would rather grep a by-default-verbose log then have to go and enable verboser logging and re-run. But sometimes you have to do that anyway (increase verbosity to debug or to trace).

Once we can get the tests to fail properly I would be more comfortable silencing them more. But I'm OK with it now too, I can always set my local level to debug or trace.

Reply:
Yes it was the http logger

So we have our test runner --silence output by default (see the jest.config.js -- `silent: true`)

This can be disabled by passing `--no-silent` to the test call you want `npm run test:unit --no-silent`

The current behavior, though, breaks jest's `silent` option entirely because it bypasses the console output.  Either way we should fix that / make sure our logger plays well with our test runner.

As far as whether --silent should be default that's fair -- I guess the default setting of silent was predicated on the idea that noise on test success means it's harder to notice when tests are failing.  I don't want to see a bunch of HTTP request logs (which are NOT pino prettied btw!) covering up the test progress / stats / etc.

Reply:
I completely agree that we will need to better integrate these and make them play well together and agree with one another, etc. It is kind of annoying that I need to lose my jest coloration or avoid my pino-pretty printing or have one silent and another loud. In other words, yes, it needs to be better.

Reply:
I know you have dug into this on another issue #59 so I'm gonna mark this thread as resolved